### DISCUSSION TITLE
Installing Platform Insteon plugin on Homebridge

### DISCUSSION THREAD

**Discussion by Jhurst4024** [kuestess/homebridge-platform-insteonlocal#236] [General]
I have been using Platform Insteonlocal on my Hoobs box but wanted to experiment with Homebridge. I install Homebridge on my Mac OS desktop computer. I then installed the Homebridge-Platform-Insteon plugin. When I click on the settings for the plugin I get the message "This plugin must be configured manually using the Homebridge UI Config Editor. Please consult the plugin documentation for instructions on how to correctly configure this plugin."

Can anyone help me on where I go from here to know what I manually need to enter to finish the install of the plugin?
And what I need to do to start using the InsteonUI?
Thanks

**Comment by kuestess** [comment]
@Jhurst4024 You may have installed the wrong plugin. There is a homebridge-platform-insteon plugin that has never been developed. You want homebridge-platform-insteonlocal

**Comment by Jhurst4024** [comment]
Thank you for moving this to discussions. You were correct. It was the old one. I installed the correct one but at the end I got several error. I attached a screen shot. I also have the log if that helps.
Thanks Joh
[Insteonlocal error.pdf](https://github.com/kuestess/homebridge-platform-insteonlocal/files/8645540/Insteonlocal.error.pdf)
n

**Comment by kuestess** [comment]
@Jhurst4024 Looks like a network error - might try uninstalling/reinstalling.

**Comment by Jhurst4024** [comment]
Thank you. Your help has been wonderful. That worked and I was able finish inputing the settings for the Insteon 2245 Hub. 
Since Homebridge is installed on iMac Desktop I assume the IP address is the one that shows on the Hombridge Status page under System Information. (see attached screen shot) which is 10.0.0.224.  So I entered on my browser 10.0.0.224:3000, but it was not able to launch or open the Insteonlocal UI. see screen shot.  Do you have any thoughts? Am I not using the correct address
![Image 5-7-22 at 2 42 PM](https://user-images.githubusercontent.com/104112968/167268034-02c5769f-b89c-4e60-8295-43211dee9986.jpg)
![Image 5-7-22 at 2 41 PM](https://user-images.githubusercontent.com/104112968/167268035-c5b32aa2-b5b2-4a4c-8ccf-12a196b3c437.jpg)
?
